preview/drm_preview.cpp

```diff
@@ -20,7 +20,7 @@ class DrmPreview : public Preview
 	~DrmPreview();
 	// Display the buffer. You get given the fd back in the BufferDoneCallback
 	// once its available for re-use.
-	virtual void Show(int fd, size_t size, int width, int height, int stride) override;
+	virtual void Show(int fd, void *mem, size_t size, int width, int height, int stride) override;
 	// Reset the preview window, clearing the current buffers and being ready to
```

Comment:
Not too fussed about this, but perhaps we should use a Span for consistency?

Reply:
Yes, I think that makes sense. Our post-processing branch has switched over to using Spans more so this will fit well (though probably cause a minor merge conflict!).